doc/debugger.rst

```diff
@@ -1388,6 +1402,27 @@ reestablish the link).
 ``msg`` is an optional string elaborating on the reason for the detach.  It may
 or may not be present depending on the nature of detachment.
 
+AppNotify notification (0x07)
+-----------------------------
+
+Format::
+
+    NFY <int: 0x07> [<tval>]* EOM
```

Comment:
It was mentioned earlier in the document that "<tval: name>" was valid metasyntax for tval-compatible values, so I figured this was okay (I left out a field name because it would just be extra noise in this context).